File: JAVA/UberBlack.java
import java.util.Map;
import java.util.ArrayList;

public class UberBlack extends Car{
    Map<String, Map<String, Integer>> typeCarAccepted;
    ArrayList<String> seatMaterial;

    public UberBlack (String license, Account driver, Integer passenger, Map<String, Map<String, Integer>> typeCarAccepted,  ArrayList<String> seatMaterial){
        super(license,driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatMaterial = seatMaterial;
    }
        
}

File: PYTHON/UberBlack.py
from car import Car

class UberPool(Car):
    typeCarAccepted = []
    seatsMateria = []

    def __init__(self, license, driver, passenger, typeCarAccepted, seatsMateria):
        super().__init__(license, driver, passenger)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMateria = seatsMateria 